changelog/README.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 16th November 2023
+
+* Extended [Get all bills](../operations/bills.md#get-all-bills) response with `AssociatedAccountIds` and [AssociatedAccountData](../operations/bills.md#bill-associated-account-data).
```

Comment:
This is more of a question than a comment; would it be better to mention we've updated Bill, instead of get all bills 🤔 There are multiple endpoints getting affected by it.. but not sure, it's up to the connectivity side. 
@VBuben any comments?